wp-content/themes/bpr2018/resources/templates/author.tpl.php

```diff
@@ -1,11 +1,13 @@
 <?php get_header(); ?>
 
 <div id="author-page" class="container-fluid">
-  <div id="author-header" class="row">
+  <div id="author-header" class="row" style="display: flex; align-items: center">
```

Comment:
It might be nice to remove the margins on the Author name since you are trying to vertically center things. The author name currently looks a bit off-center.